### DISCUSSION TITLE
Looking Support for Winbond W25Q64JVFSIQ & Cfeon EN25QX128A

### DISCUSSION THREAD

**Discussion by xiv3r** [nofeletru/UsbAsp-flash#96] [Q&A]
Looking Support for EEPROM IC

Winbond W25Q64JVFSIQ
Cfeon EN25QX128A

**Comment by nofeletru** [comment]
W25Q64JV already supported.
EN25QX128A added to chiplist [https://github.com/nofeletru/UsbAsp-flash/raw/master/chiplist.xml](https://github.com/nofeletru/UsbAsp-flash/raw/master/chiplist.xml)

**Reply by xiv3r** [reply]
EN25X128A work fine but the W25Q64JV can't detect W25Q64JVFSIQ...
